packages/paste-website/src/pages/tokens/overview/index.mdx

```diff
@@ -0,0 +1,171 @@
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/tokens/overview/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<NormalizeComponentHeader
+  description={props.pageContext.frontmatter.description}
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.TOKENS}
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Introduction
+
+Design tokens make up the smallest base parts of a design language and help distribute it across different platforms and technologies. This helps to create a more seamless user experience across products and digital properties, no matter how they are built, by allowing that visual design language to be shared from a single source of truth.
+
+As the attributes of a design language are never hard-coded into any file, an evolution of the design language can be made with fewer updates to design assets and products by just updating the token values from that single central source.
+
+All Paste components use design tokens. Additionally, [styling primitives with tokens unlocks the ability to create custom UI](/customization/composing-custom-components-with-design-tokens) that still uses Paste.
+
+<Img fluid={props.data.inputTokens.childImageSharp.fluid} />
+<ImageCaption>
+  For example, an Input in an error state is styled using a number of tokens that set styles such as border color and
+  thickness, text color and size, padding, and margin.
+</ImageCaption>
+
+<Callout>
+  <CalloutTitle as="h3">Need a token that doesn't exist in Paste yet?</CalloutTitle>
+  <CalloutText>
+    We add new tokens every so often when there’s a strong semantic purpose that can exist across multiple products. If
+    you think you have a need for a new token, please{' '}
+    <Anchor href="https://github.com/twilio-labs/paste/discussions" showExternal>
+      start a GitHub discussion
+    </Anchor>
+    .
+  </CalloutText>
+</Callout>
+
+## Token Naming
+
+Design tokens represent fundamental decisions of Paste’s design language, specifying their intended usage with their semantic naming in a predictable and clear way. For example, **`$color-text-link-destructive`** is used for displaying link text that communicates an action is destructive. Using the appropriate token ensures that all destructive actions are styled consistently, appearing as red text in this case. Once a token name is created, it can never change, or be removed.
+
+Tokens are named semantically to specify their intended usage. The format of a design token name is:
+**`$type-category-instance-variant-state`**
+
+<DoDont>
+  <Do title="Do" body="Name a token based on its use case or purpose." center>
+    <Text fontFamily="fontFamilyCode">$color-background-success</Text>
+  </Do>
+  <Dont title="Don't" body="Name a token based on its value." center>
```

Comment:
suggestion: I find it most clear to put "Don't" in the beginning of the body text here, even though it's repetitive, so this could be "Don't name a token based on its value".  I think we have this added in other pages as well. The way it is right now is still correct though, so it's definitely not blocking (and borderline nitpick-y)